src/Layers/xrRenderPC_GL/gl_rendertarget_draw_rain.cpp

```diff
@@ -147,12 +147,11 @@ void CRenderTarget::draw_rain(light& RainSetup)
         // clouds xform
         Fmatrix m_clouds_shadow;
         {
-            static float w_shift = 0;
             Fmatrix m_xform;
             //Fvector			direction	= RainSetup.direction	;
             Fvector normal;
             normal.setHP(1, 0);
-            //w_shift		+=	0.003f*Device.fTimeDelta;
+            //static float w_shift = 0.003f*Device.fTimeDelta;
```

Comment:
You changed the original code logic, please, leave the variable in the original place. Commented, though)